src/components/GridItem.vue

```diff
@@ -377,7 +377,15 @@
             },
             maxW: function () {
                 this.tryMakeResizable();
-            }
+            },
+            "$parent.margin": function (margin) {
+                if (!margin || (margin[0] == this.margin[0] && margin[1] == this.margin[1])) {
+                    return;
+                }
+                this.margin = margin.map(m => parseInt(m));
```

Comment:
why are you using `parseInt`? For example, in my case, reactive margins are calculated values

I think you need to use `Number(m)` instead of `parseInt(m)`.

Reply:
I think that `parseInt(...)` should be the same as `Number(...)` when the parameter is a number while use `parseInt(...)` will also work if the parameter is a string.

Which error did you encounter when using `parseInt(...)` ?

Reply:
@yfwz100, No, i didn't mean happening error. I encountered wrong result with `parseInt`, because margins were calculated value like `[10 * ratio, 10 * ratio]; `. 

`const ratio = dynamicSize / STATIC_SIZE`

Reply:
@yfwz100 what @turdiyev means is the margin should support float numbers too. For example:

`
const ratio = 0.75;
parseInt(10*ratio); // 7 
Number(10*ratio); // 7.5
`

Reply:
Yep, `parseInt(m)` converts to integer type, `Number(m)` or `+m` converts float type.

Reply:
I understand... Thanks for the reply! 👍